packages/vscode-extension/src/project/deviceSession.ts

```diff
@@ -59,6 +60,7 @@ export class DeviceSession implements Disposable {
 
     progressCallback(StartupMessage.AttachingDebugger);
     await this.startDebugger();
+    isNodeModulesInstalled("npm");
```

Comment:
Why we specify npm here? Also, this does nothing except slowing down run procedure.

Reply:
remanence of debbuging process